ProcessMaker/Http/Controllers/Api/ProcessRequestController.php

```diff
@@ -164,10 +162,14 @@ public function index(Request $request, $getTotal = false, User $user = null)
             } elseif ($request->input('total') == 'true') {
                 return ['meta' => ['total' => $query->count()]];
             } else {
-                $response = $query->orderBy(
-                    str_ireplace('.', '->', $request->input('order_by', 'name')),
-                    $request->input('order_direction', 'ASC')
-                )
+                $request->input('order_by') == 'process.alternative'
```

Comment:
Please, let's use the **if** structure. In the future, we may need to add other columns, so future changes will be clearer and easier to understand.